### PR TITLE
fix definition of Geo segment

### LINKED ISSUE #1134: fix definition of GeoSegment
The definition of GeoSegment appears to have a typo. It says a GeoSegment is a single portion of a _GeoRegion_ instead of being a single portion of a _GeoRoute_.

Current definition of GeoSegment: "A single portion of a GeoRegion which has been divided (i.e., segmented)."

Proposed: "A single portion of a GeoRoute consisting of consecutive GeoPoints belonging to the GeoRoute."

Proposed addition to GeoRoute: "Each consecutive pair of GeoPoints in a GeoRoute is a GeoSegment."

### PR DESCRIPTION
**Pull request by philblackwood**
closes #1134

### CODE REVIEW

File: ontologies/gistCore.ttl [hunk 1]
```diff
@@ -778,6 +778,7 @@ gist:GeoRoute
 	] ;
 	skos:definition "An ordered set of GeoPoints that defines a path from starting point to ending point."^^xsd:string ;
 	skos:prefLabel "Geo Route"^^xsd:string ;
+	skos:scopeNote "Each pair of consecutive GeoPoints in a GeoRoute is a GeoSegment."^^xsd:string ;
```
Comment (uscholdm) [inline ontologies/gistCore.ttl:781]: Add to scope note tha A single GeoSegment also a GeoRoute. If it's not now, then probably it should be a subclass.

**Review by uscholdm — APPROVED** [review]

File: ontologies/gistCore.ttl (hunk 1 above)
Comment (rjyounes) [reply ontologies/gistCore.ttl:781]: I agree that GeoSegment is a subclass of GeoRoute with two members, but we are using two different mechanisms to define them. In the case of GeoSegment, we are using comesFrom/goesToPlace, whereas we are using the ordered collection mechanism for GeoRoutes. They don't need to be mutually exclusive, but the way they are currently defined they don't harmonize well. This is considered in issue #1165.

**Review by rjyounes — APPROVED** [review]

**Comment by MichaelSullivanArchitect** [comment]
seems to me the definition should possibly change to something like "An ordered set of GeoSegments..."

**Comment by uscholdm** [comment]
> seems to me the definition should possibly change to something like "An ordered set of GeoSegments..."

I think the definition is fine as it is. It may be the case that as a consequence, it is also "An ordered set of GeoSegments...". If so, it could be added as a scope note, if we think it would be helpful.

**Pull request merged**